ModelBase/src/nodes/Reference.cpp

```diff
@@ -85,7 +85,8 @@ bool Reference::resolveHelper(bool indirect)
 
 	auto newTarget = computeTarget();
 
-	Q_ASSERT(!newTarget || (newTarget->definesSymbol() && newTarget->symbolName() == name_));
+	//TODO@cyril What is this for?
+//	Q_ASSERT(!newTarget || (newTarget->definesSymbol() && newTarget->symbolName() == name_));
```

Comment:
Looking at what it does, I believe using these macros means that the referencedNode will be added to the ReferenceNode as a child (meaning that we would have multiple parents at some point)? So it seems we should still go with a manual way of writing the code, or am I wrong there?